filters/code_mixing/filter.py

```diff
@@ -0,0 +1,217 @@
+    """
+
+    tasks = [
+        TaskType.E2E_TASK,
+        TaskType.TEXT_TAGGING,
+        TaskType.TEXT_CLASSIFICATION,
+    ]
+    languages = [
+        "en",
+        "ru",
+        "de",
+        "fr",
+        "it",
+        "ja",
+        "es",
+        "ceb",
+        "tr",
+        "pt",
+        "uk",
+        "eo",
+        "pl",
+        "sv",
+        "nl",
+        "he",
+        "zh",
+        "hu",
+        "ar",
+        "ca",
+        "fi",
+        "cs",
+        "fa",
+        "sr",
+        "el",
+        "vi",
+        "bg",
+        "ko",
+        "no",
+        "mk",
+        "ro",
+        "id",
+        "th",
+        "hy",
+        "da",
+        "ta",
+        "hi",
+        "hr",
+        "sh",
+        "be",
+        "ka",
+        "te",
+        "kk",
+        "war",
+        "lt",
+        "gl",
+        "sk",
+        "bn",
+        "eu",
+        "sl",
+        "kn",
+        "ml",
+        "mr",
+        "et",
+        "az",
+        "ms",
+        "sq",
+        "la",
+        "bs",
+        "nn",
+        "ur",
+        "lv",
+        "my",
+        "tt",
+        "af",
+        "oc",
+        "nds",
+        "ky",
+        "ast",
+        "tl",
+        "is",
+        "ia",
+        "si",
+        "gu",
+        "km",
+        "br",
+        "ba",
+        "uz",
+        "bo",
+        "pa",
+        "vo",
+        "als",
+        "ne",
+        "cy",
+        "jbo",
+        "fy",
+        "mn",
+        "lb",
+        "ce",
+        "ug",
+        "tg",
+        "sco",
+        "sa",
+        "cv",
+        "jv",
+        "min",
+        "io",
+        "or",
+        "as",
+        "new",
+        "ga",
+        "mg",
+        "an",
+        "ckb",
+        "sw",
+        "bar",
+        "lmo",
+        "yi",
+        "arz",
+        "mhr",
+        "azb",
+        "sah",
+        "pnb",
+        "su",
+        "bpy",
+        "pms",
+        "ilo",
+        "wuu",
+        "ku",
+        "ps",
+        "ie",
+        "xmf",
+        "yue",
+        "gom",
+        "li",
+        "mwl",
+        "kw",
+        "sd",
+        "hsb",
+        "scn",
+        "gd",
+        "pam",
+        "bh",
+        "mai",
+        "vec",
+        "mt",
+        "dv",
+        "wa",
+        "mzn",
+        "am",
+        "qu",
+        "eml",
+        "cbk",
+        "tk",
+        "rm",
+        "os",
+        "vls",
+        "yo",
+        "lo",
+        "lez",
+        "so",
+        "myv",
+        "diq",
+        "mrj",
+        "dsb",
+        "frr",
+        "ht",
+        "gn",
+        "bxr",
+        "kv",
+        "sc",
+        "nah",
+        "krc",
+        "bcl",
+        "nap",
+        "gv",
+        "av",
+        "rue",
+        "xal",
+        "pfl",
+        "dty",
+        "hif",
+        "co",
+        "lrc",
+        "vep",
+        "tyv",
+    ]
+
+    def __init__(self, k=2, threshold=0.5):
+        super().__init__()
+        self.nlp = spacy.load("en_core_web_sm")
```

Comment:
You might want to use spacy like [this](https://github.com/GEM-benchmark/NL-Augmenter/blob/d39f0e5f57a79937296351996617ac03ee80fb74/transformations/close_homophones_swap/transformation.py#L55).